python/tvm/build_module.py

```diff
@@ -387,6 +387,7 @@ def lower(sch,
     binds, arg_list = get_binds(args, compact, binds)
 
     # Phase 1
+    stmt = ir_pass.TensorCore(stmt, sch, binds)
```

Comment:
We need to check current target is cuda before calling this

Reply:
Thanks! we'll add that.

Reply:
Done.